cmd/pairing/agent.go

```diff
@@ -70,3 +82,19 @@ func agentRegisterF(command *cobra.Command, args []string) error {
 	fmt.Printf("will be associated permanently to the Device and it won't be changeable anymore.\n")
 	return nil
 }
+
+func agentUnregisterF(command *cobra.Command, args []string) error {
```

Comment:
Add a prompt for this, with a non-interactive option (you can find every helper in astartectl cluster). We don't want users to accidentally hit this.